### PR TITLE
Feature/organization

### PR DESCRIPTION
**Pull request by florianow**

### CODE REVIEW

File: foundations/likvid-prod/platforms/aws/README.md [hunk 1]
```diff
@@ -4,30 +4,30 @@ aws:
...
+In this AWS account our meshstack dev installation is running.
+
+```text
+Account Name: meshstack-dev
+Root User Email: meshstack-dev@meshcloud.io
+```
+
+Password for this account can be found in pass in `dev/locations/aws/meshstack-dev`.
+
+The infrastructure is deployed in `Frankfurt` AWS region. Remember to switch to that region to find the running infrastructure.
```
Comment (malhussan) [inline foundations/likvid-prod/platforms/aws/README.md:23]: d: This is outdated when dev infra was running in AWS, I think we can omit these lines (14-23)

File: foundations/likvid-prod/platforms/aws/README.md [hunk 2]
```diff
@@ -4,30 +4,30 @@ aws:
...
+
+## Global Infrastructure
+
+> TODO: this is no longer current since the docs have moved on to netlify?
+
+This is also the account that hosts [docs.meshcloud.io](http://docs.meshcloud.io) and [docs.dev.meshcloud.io](http://docs.dev.meshcloud.io) buckets.
```
Comment (malhussan) [inline foundations/likvid-prod/platforms/aws/README.md:29]: d: Also probably not relevant for Likvid, but is internal

File: kit/aws/landingzones/cloud-native/documentation.tf [hunk 3]
```diff
@@ -0,0 +1,10 @@
+output "documentation_md" {
+  value = <<EOF
+# Cloud Native Landing Zone
+
+This landing zone is inteded for applications teams that want to work natively without AWS and can take
```
Comment (malhussan) [inline kit/aws/landingzones/cloud-native/documentation.tf:5]: q: what does it mean without AWS in this context?

**Review by malhussan — APPROVED** [review]
lgtm, just some minor comments

File: kit/aws/landingzones/cloud-native/documentation.tf (hunk 3 above)
Comment (florianow) [reply kit/aws/landingzones/cloud-native/documentation.tf:5]: i dont know :), I copied it from meshcloud-dev...maybe it should mean "with" and not "without"...

**Pull request merged**